src/main/java/taskmaster/task/Task.java

```diff
@@ -0,0 +1,74 @@
+package taskmaster.task;
+
+/**
+ * This class encapsulates the information necessary for a user task.
+ */
+
+
+public abstract class Task {
+    /** Name of the task. **/
+    protected String taskName;
+
+    /** Status of the task. **/
+    protected Boolean isCompleted;
```

Comment:
I think you should use `boolean` instead of `Boolean` since it is much more efficient

Reply:
Thanks for raising this up! Didn't notice I used the wrapper class here.